repo/packages/D/dcos-enterprise-cli/10/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-enterprise-cli",
+  "version": "1.0.7",
```

Comment:
The binary itself isn't different, its the zip folder format. In this case I'm going to keep the version because I think the same CLI code should be the same version.